src/shaders/symbol_icon.vertex.glsl

```diff
@@ -1,22 +1,25 @@
-
 attribute vec4 a_pos_offset;
+attribute vec2 a_label_pos;
 attribute vec4 a_data;
 
 // icon-size data (see symbol_sdf.vertex.glsl for more)
 attribute vec3 a_size;
 uniform bool u_is_size_zoom_constant;
 uniform bool u_is_size_feature_constant;
-uniform mediump float u_size_t; // used to interpolate between zoom stops when size is a composite function
-uniform mediump float u_size; // used when size is both zoom and feature constant
-uniform mediump float u_layout_size; // used when size is feature constant
+uniform highp float u_size_t; // used to interpolate between zoom stops when size is a composite function
+uniform highp float u_size; // used when size is both zoom and feature constant
+uniform highp float u_layout_size; // used when size is feature constant
+uniform highp float u_camera_to_center_distance;
+uniform highp float u_pitch;
+uniform highp float u_collision_y_stretch;
```

Comment:
I don't think it's necessary (although I did most of my testing with highp precision, so maybe lowering the precision requirement would expose some problems). I ended up standardizing on highp because I didn't have any better criteria for choosing precision and having a jumble of lowp/mediump/highp just seemed to make the shader code a little bit more confusing. @jfirebaugh pointed me at #2096 as a guideline.